node-placeholder-scaler/scaler/scaler.py

```diff
@@ -4,6 +4,7 @@
 import subprocess
 import tempfile
 import time
+import json
```

Comment:
i'm curious why you're adding json when we're already parsing a bunch of stuff as yaml...  since we're already doing most of our data processing w/yaml i'd prefer that we stick to it, vs having json *and* yaml.

this is just a nitpick, and not a hill i'm planning to die on.  :)

Reply:
nvm i see where it's used